Utilities/constants.py

```diff
@@ -30,6 +29,12 @@ class Drones(Enum):
 RANGEFINDER_EPSILON = 0.03
 DEFAULT_ARM_TIMEOUT = 60
 
+# Values based on heapq, with is a min-heap.
+class Priorities(Enum):
+    LOW = 3
+    MEDIUM = 2
+    HIGH = 1
+
 # DroneKit Vehicle Modes
```

Comment:
Make this a docstring under the class, otherwise this information will not appear in typical intellisense parsers.